paper/src/main/java/com/untamedears/realisticbiomes/growth/FungusGrower.java

```diff
@@ -55,6 +55,7 @@ public void setStage(final Plant plant, final int stage) {
 		final ServerLevel world = ((CraftWorld) block.getWorld()).getHandle();
 		final BlockPos position = new BlockPos(block.getX(), block.getY(), block.getZ());
 		//Taken from CraftWorld.generateTree()
+		block.setType(Material.AIR);
```

Comment:
Are we sure you can't remove reinforcements with this?

Reply:
Plants cannot be reinforced